ThemeDecorator/ThemeDecorator.js

```diff
@@ -257,8 +257,8 @@ const ThemeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	const defaultSkin = 'gallium';
 
 	const bgClassName = classnames(
-		'enact-fit',
 		{
+			'enact-fit': !disableFullscreen,
```

Comment:
Let's move braces to the same line with adjusted parentheses so clean up indentations here.